autoload/db_ui/schemas.vim

```diff
@@ -85,11 +85,52 @@ let s:mysql = {
       \ 'layout_flag': '\\G',
       \ 'quote': 0,
       \ }
+
+let s:oracle_args = 'echo "'.join(
+      \    [
+           \  'SET linesize 4000',
+           \  'SET pagesize 4000',
+           \  'COLUMN owner FORMAT a20',
+           \  'COLUMN table_name FORMAT a25',
+           \  'COLUMN column_name FORMAT a25',
+           \  '%s',
+      \    ],
+      \    ";\n"
+      \ ).';" |'
+let s:oracle_foreign_key_query = "
```

Comment:
I didn't bother formatting these with indentation or newlines (other than from `s:oracle_args` since users can't see them. I did format the table helpers though.